### PR TITLE
Replace kafka publish by webhooks

### PR DESCRIPTION
**Pull request by quaxsze**

### CODE REVIEW

File: udata/core/dataset/events.py [hunk 1]
```diff
@@ -40,35 +39,34 @@ def serialize_resource_for_event(resource):
...
+    r = requests.post(url, json=payload)
+    r.raise_for_status()
 
 
 @Dataset.on_resource_added.connect
 def publish_added_resource_message(sender, document, **kwargs):
-    if current_app.config.get('PUBLISH_ON_RESOURCE_EVENTS'):
-        publish.delay(document, kwargs['resource_id'], KafkaMessageType.CREATED)
+    if current_app.config.get('PUBLISH_ON_RESOURCE_EVENTS') and current_app.config.get('RESOURCES_ANALYSER_URI'):
+        publish.delay(f"{current_app.config.get('RESOURCES_ANALYSER_URI')}/api/resource/created/", document, kwargs['resource_id'], KafkaMessageType.CREATED)
```
Comment (maudetes) [inline udata/core/dataset/events.py:60]: Not sure we need the `KafkaMessageType` anymore?

File: udata/core/dataset/events.py [hunk 2]
```diff
@@ -40,35 +39,34 @@ def serialize_resource_for_event(resource):
...
-    produce(
-        kafka_uri=current_app.config.get('KAFKA_URI'),
-        topic=get_topic(message_type),
-        service='udata',
-        key_id=str(resource_id),
-        document=resource,
-        meta={'message_type': message_type, 'dataset_id': str(document.id)}
-    )
+    payload = {
+        'key': str(resource_id),
```
Comment (maudetes) [inline udata/core/dataset/events.py:49]: I'm not sure about this payload. `key` doesn't make much sense now.
Maybe we should have something like
```
{
  'resource_id': '...',
  'dataset_id': '...',
  'document': '...'
}
```

**Review by maudetes — COMMENTED** [review]
Seems to work nicely!

I have some suggestions on the payload

**Comment by quaxsze** [comment]
Done in https://github.com/opendatateam/udata/pull/2783

**Pull request closed without merging**